packages/dx-grid-core/src/plugins/table-column-visibility/helpers.test.js

```diff
@@ -0,0 +1,34 @@
+import { TABLE_DATA_TYPE } from '../table/constants';
+import { isEmptyMessageShow } from './helpers';
+
+describe('TableColumnVisibility helpers', () => {
+  describe('#isEmptyMessageShow', () => {
+    it('should show empty message when all columns are hidden', () => {
+      const grouping = [
+        { columnName: 'a' },
+        { columnName: 'b' },
+      ];
+
+      expect(isEmptyMessageShow(grouping, []))
```

Comment:
`isEmptyMessageShow` takes only one argument - `tableColumns`. But here there are two args. And one of them is called `grouping`.

And let's rename `isEmptyMessageShow`. `showEmptyMessage` or `tableColumnsExist`.